### DISCUSSION TITLE
How can I get top languages card on my profile?

### DISCUSSION THREAD

**Discussion by voyager19878** [community/community#21245] [Other Feature Feedback, Questions, & Ideas] [answered]
I'm newbie to GitHub. How can I get top languages card on my profile?

I will be glad if you help me. :octocat:

**Comment by hud0shnik** [comment, marked as answer]
Create ur personal readme and add this 👇

```
<img src="https://github-readme-stats.vercel.app/api/top-langs/?username=voyager19878"/>
```

**Reply by voyager19878** [reply]
TY :smile:

**Reply by hud0shnik** [reply]
And read this https://github.com/github-community/community/discussions/19142

**Reply by hud0shnik** [reply]
U can also copy cards from ![here](https://github.com/hud0shnik/hud0shnik).
